review_management.md

```diff
@@ -140,13 +140,44 @@ When talking to the proposal authors, you should be clear about the role you’r
+
+A vision document consists primarily of three sections: an analysis of the current state of the area, a definition of the goals that evolution in this area should strive to satisfy, and a description of the general approach that should be taken by evolution.
+
+The analysis section should endeavor to present an objective and accurate view of the current state of affairs.  How do users get work done in this area?  What problems do they encounter?  Can these problems be solved within the Swift project, or are users forced to turn outside of it?  Are there systemic problems, and are they exacerbated by the solutions users are forced to use?
+
+The goals section should define and analyze the goals that any evolution in this area must satisfy.  These should not be reduced to simple aspirational statements.  How do the specific problems faced in this area relate to the overall goals of the Swift project?  What are the most important problems to solve?  Different goals (such as safety and performance) can often be in conflict with each other; how should these trade-offs be resolved?
+
+The approach section should lay out a high-level plan for the work that should be done.  It should start by laying out the basic principles of the approach, then describe how those principles can be concretely applied to specific problems in the area.  Often this will take the form of brief descriptions of a number of different proposed features.  These proposal descriptions do not need to be intensely detailed and may leave major questions open for future discussion; of all the sections, it's most okay for this section to sometimes be vague or incomplete, because all of the content will be separately reviewed.  That said, the basic principles of the approach should be clearly laid out and convincingly argued for.
+
+Vision documents must be approved by the appropriate evolution workgroup(s).  (There was no expectation of this for the old "manifestos", although many were written by Core Team members and so were understood to carry a certain weight.)  Approving a vision signals strong agreement with the analysis and goals sections, as well as the basic principles of the approach.  It does not endorse any of the details of the proposals, and the workgroup should not feel bound in any decision by having previously accepted a vision document.  The evolution community should treat the approval of a vision as a broad statement about how the workgroup thinks about the subject area, not as a prejudgment on any related proposal that arises in the future.
```

Comment:
```
Vision documents must be approved by the appropriate evolution workgroup(s).
```
Should we state explicitly that vision documents aren't reviewed? We say that for roadmaps, but I don't think we do for visions

Reply:
```suggestion
Vision documents must be approved by the appropriate evolution workgroup(s).  (There was no expectation of this for the old "manifestos", although many were written by Core Team members and so were understood to carry a certain weight.)  Approval is purely a decision by the workgroup and does not go through the open evolution process.  Approving a vision signals strong agreement with the analysis and goals sections, as well as the basic principles of the approach.  It does not endorse any of the details of the proposals, and the workgroup should not feel bound in any decision by having previously accepted a vision document.  The evolution community should treat the approval of a vision as a broad statement about how the workgroup thinks about the subject area, not as a prejudgment on any related proposal that arises in the future.
```